backend/routers/spiritual.py

```
@@ -789,4 +792,100 @@ def get_default_lord(house_index: int) -> str:
+                "needs_birth_details": True
+            }
+        
+        # If birth chart data exists and is not expired, return it
+        if user_data['birth_chart_data']:
+            try:
+                chart_data = json.loads(user_data['birth_chart_data']) if isinstance(user_data['birth_chart_data'], str) else user_data['birth_chart_data']
+                return {
+                    "success": True,
+                    "complete_profile": chart_data
+                }
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)